src/libraries/System.ServiceProcess.ServiceController/src/System.ServiceProcess.ServiceController.csproj

```diff
@@ -89,6 +89,9 @@
     <Reference Include="System.Threading.Thread" />
     <Reference Include="System.Threading.ThreadPool" />
   </ItemGroup>
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netcoreapp3.1'))">
+    <Reference Include="System.ComponentModel.TypeConverter" />
```

Comment:
Why is that reference only needed on netcoreapp3.1 and not on net6.0?

Reply:
If I leave that out, I get:
```
\runtime\src\libraries\System.ServiceProcess.ServiceController\src\System\ServiceProcess\ServiceController.cs(16,6): error CS0246: The type or namespace name 'DesignerAttribute' could not be found (are you missing a using directive or an assembly reference?)
\runtime\src\libraries\System.ServiceProcess.ServiceController\src\System\ServiceProcess\ServiceController.cs(16,6): error CS0246: The type or namespace name 'Designer' could not be found (are you missing a using directive or an assembly reference?)
```

Reply:
Probably because we moved those types over into another assembly. Thanks for checking.